EjercicioDiaDos.cs

```diff
@@ -37,52 +35,39 @@ public void llamarEjercicioUno()
         public void llamarEjercicioDos()
         {
             Console.WriteLine("Digite un Número: ");
-            int numero = int.Parse(Console.ReadLine());
-
-            EjercicioDiaDos ejercicioDia = new EjercicioDiaDos();
+            int numero = Convert.ToInt32(Console.ReadLine());
 
-            numero = ejercicioDia.DeterminarNumero(numero);
+            string determinenumero = ejercicioDia.DeterminarNumero(numero);
 
-                Console.WriteLine("Número es Negativo" + (numero < 0));
-                Console.WriteLine("Número es Positivo" + (numero >= 0));
+            if (numero >= 0)
+                Console.WriteLine("Número es Positivo");
+            else
+                Console.WriteLine("Número es Negativo");
 
             Console.ReadKey();
         }
 
         public void llamarEjercicioTres()
         {
-            Console.WriteLine("Entre un Número");
-            int numero = Int32.Parse(Console.ReadLine());
+            string numerodivisible;
 
-            EjercicioDiaDos ejercicioDia = new EjercicioDiaDos();
+            Console.WriteLine("Entre un Número");
+            int multiplo = int.Parse(Console.ReadLine());
 
-            numero = Int32.Parse(ejercicioDia.CalculoDeMultiplos(numero));
+            numerodivisible = ejercicioDia.CalculoDeMultiplos(multiplo);
 
-            Boolean numeroEsDivisible = false;
-            if (numero % 2 == 0)
-            {
+            if (multiplo % 2 == 0 && multiplo % 3 == 0)
```

Comment:
Porque esto esta acá? esta logica debe estar en el metodo CalculoDeMultiplos, estas repitiendo exactamente lo mism: el ejercicio dice que se debe retornar un string con el mensaje de su operacion.

```suggestion
           Console.WriteLine(numerodivisible);
```

Reply:
Cambios realizados a punto

Reply:
Realizado